LEAF_Request_Portal/sources/System.php

```diff
@@ -289,413 +264,153 @@ public function updateGroup(int $groupID): array
     /**
      * @param int $groupID
      *
-     * @return array
-     *
-     * Created at: 6/30/2023, 1:25:07 PM (America/New_York)
+     * @return string
      */
-    private function updateCatPrivs(int $groupID): array
```

Comment:
Just back-checking.  Are we sure that nothing is going to be broken by this "old" function going away?

Reply:
This just got moved down the page, it's still here, just not in this location.